akka-http-core/src/main/scala/akka/http/scaladsl/ConnectionContext.scala

```diff
@@ -120,31 +118,30 @@ private[http] case class DeprecatedSslContextParameters(
  * This constructor is INTERNAL API, use ConnectionContext.https instead
  */
 @InternalApi
-final class HttpsConnectionContext(
-  val sslContextData:                                                                  Either[DeprecatedSslContextParameters, Option[(String, Int)] => Engine],
-  @deprecated("here for binary compatibility", since = "10.2.0") val sslConfig:        Option[AkkaSSLConfig]                                                   = None,
```

Comment:
Converted those from fields to `def` that propagate to the left values (or fail otherwise).